### ISSUE TITLE
Control window illuminated sign on space state

### ISSUE THREAD

**Issue by sjefferson99** [somakeit/smib#193]
Also support times not to illuminate regardless of state.

**Comment by samcorky** [comment]
Can have the times configurable on Slack. I think time management should happen from smib rather than smibhid. Smibhid can use smib as a ntp server

**Comment by sjefferson99** [comment]
Yes, this would be a smib feature, smibhid would only be setting the state with smib.

**Comment by samcorky** [comment]
this is WLED so has existing timing functionality and API.

**Comment by samcorky** [comment]
Async python [WLED library](https://github.com/frenck/python-wled)

**Comment by sjefferson99** [comment]
@samcorky So is this a driver upgrade for smibhid to support WLED output, but take configurable times of operation to AND with space state driven by SMIB. So maybe an API endpoint for setting times, this would probably take a device field in the payload so could also configure relay etc. This would depend on getting smibhid able to write its own json config which has been hanging out as a nice to have issue.

**Comment by samcorky** [comment]
I was thinking that this would be driven by SMIB, as no extra hardware is needed. Unless there is a reason I am missing for it to be SMIBHID controlled?

**Comment by sjefferson99** [comment]
It's in another room and you'd need to run a really long wire, other than that you could drive it from smib. It would be unusual in that all the other components are driven from smibhid (fan, light buttons, front sign light by relay idea), but it might be smib is more suited for that library. I've not looked into it.

**Comment by samcorky** [comment]
The WLED sign is connected to the SoMakeIt wifi and exposes an HTTP API; no need for a wire!

**Comment by sjefferson99** [comment]
I see, I thought it was a standalone microcontroller. Carry on.
